### PR TITLE
Lido app/marcus/filter

### PR DESCRIPTION
**Pull request by marhong2**

### CODE REVIEW

**Comment by marhong2** [comment]
It should be good now @tyleradams

File: test/formulas/reference/testcase.js [hunk 1]
```diff
@@ -34,6 +34,18 @@ module.exports = {
         'COLUMNS({1,2,3})': FormulaError.VALUE,
         'COLUMNS("A1")': FormulaError.VALUE
     },
+    FILTER: {
+        'SUM(FILTER(A2,A2:A10,B2:B10))' : .69,
```
[the remark this answers is not in the document]
Comment (marhong2) [reply test/formulas/reference/testcase.js:38]: For the original excel sheet the format is : FILTER(RETURN_ARRAY, LOOKUP_ARRAY = LOOKUP_VALUE). I got errors when I tried the LOOKUP_ARRAY = LOOKUP_VALUE, so I split it into two inputs, LOOKUP_ARRAY and LOOKUP_VALUE. I figured it was the easiest fix.

**Comment by marhong2** [comment]
Updated PR @tyleradams with your changes implemented. I also re-ordered my parameters to more closely match the Excel's ordering. I also double checked if I can format the inputs as RANGE=LOOKUP_VALUE (e.g. A1:A10=B10) and it fails. Specifically, when I print the parameter received in the FILTER function, I get null.

File: test/formulas/reference/testcase.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (tyleradams) [reply test/formulas/reference/testcase.js]: It's not. It's `FILTER(RETURN_ARRAY,CONDITION_ARRAY,DEFAULT)`

**Pull request closed without merging**